src/cirrus/lib/statedb.py

```diff
@@ -397,12 +403,13 @@ def set_aborted(self, payload_id: str) -> str:
         logger.debug("set aborted")
         return response
 
-    def query(self, collections_workflow: str, state: str=None, since: str=None,
+    def query(self, collections_workflow: str, error_begins_with: str = None, state: str=None, since: str=None,
```

Comment:
```suggestion
    def query(self, collections_workflow: str, error_begins_with: str=None, state: str=None, since: str=None,
```

Reply:
It seems less than ideal to add this in the middle of the args list. Doing so can break backward compatibility if any callers are using positional arguments for these optional args, as unless explicitly specified all args can be passed by position or kwarg. Here's a stackoverflow answer with some additional context on arg types: https://stackoverflow.com/a/57819001.

In short, I'd move this new arg to the end of the args list.